vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/util/proxy/dial_test.go

```diff
@@ -96,7 +96,7 @@ func TestDialURL(t *testing.T) {
 			ts.TLS = &tls.Config{Certificates: []tls.Certificate{cert}}
 			ts.StartTLS()
 
-			tlsConfigCopy := utilnet.CloneTLSConfig(tc.TLSConfig)
+			tlsConfigCopy := tc.TLSConfig
```

Comment:
@liggitt it also looks like the copy bug is still present in upstream as well: https://github.com/kubernetes/kubernetes/blob/53a5ee7929320056210876b521f861227a4328d6/staging/src/k8s.io/apiserver/pkg/util/proxy/dial_test.go#L99